src/reynir_correct/errtokenizer.py

```diff
@@ -2621,16 +2729,53 @@ def late_fix_merges(
             and isinstance(token, CorrectToken)  # type: ignore
             and not token.error
         ):
-            token.set_error(
-                SpellingError(
-                    "005",
-                    "Rita á '{}' sem '{}'".format(
-                        token.original.strip(), token.txt
-                    ).strip(),
-                    original=token.original,
-                    suggest=token.txt,
-                ),
-            )
+            if token.error_code in ignore_rules or "S005" in ignore_rules:
+                token.remove_error(token.original)
+            else:
+                token.set_error(
+                    SpellingError(
+                        "005",
+                        "Rita á '{}' sem '{}'".format(
+                            token.original.strip(), token.txt
+                        ).strip(),
+                        original=token.original,
+                        suggest=token.txt,
+                    ),
+                )
+        elif (
+            isinstance(token, CorrectToken)  # type: ignore
+            and token.error
+            and " " in token.txt
+            and token.error_suggest
+            and not " " in token.error_suggest
+            and token.original
+        ):
+            # Error was added to the token before merging with another (fimm milljónir *kóna*)
+            # Capitalization errors are handled elsewhere, most likely spelling errors here
+            # Determine which word(s) in the string the error represents
+            suggest: List[str] = []
+            original: List[str] = []
+            for word in token.txt.split(" "):
+                if word == token.error_original:
+                    suggest.append(token.error_suggest)
+                    original.append(token.error_original)
+                else:
+                    suggest.append(word)
+                    original.append(word)
+            if token.error_code in ignore_rules or "S005" in ignore_rules:
```

Comment:
Move this check to the if statement above? This is a bit kludgy as is.

Reply:
This is an attempt at isolating the switch. I don't want the switch to affect the overall if flow, i.e. I want the same if-clause to catch it, to prevent later if-clauses catching it by mistake with unintended consequences. Also, I need the original value created to remove the error, and want to update the error in the else-clause. This would probably be even more kludgy if I attempted to split this up.